spec/commands/show_spec.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-RSpec.describe "bundle show" do
+RSpec.describe "bundle show", :bundler => "< 3" do
```

Comment:
`bundle show` should be tested on bundler 3.

Reply:
the `show` command is removed in Bundler 3, so i'm not sure why that should continue to be tested.

Reply:
You're right.

I recall seeing some `show` stuff that's not supported on any of the alternatives (`info` or `list`), so I think there's a bit more work to do for this deprecation. But yeah, being the plan to remove `show` in bundler 3, this spec can be skipped there :+1:.

Reply:
I reviewed the latter in #7026.